.github/workflows/platform.dependencies.yml

```diff
@@ -22,16 +22,22 @@ on:
   #     - '.github/workflows/platform.dependencies.yml'
   #     - 'utilities/pipelines/dependencies/**'
 
+permissions:
+  id-token: write # OIDC
+  contents: read # OIDC
+  checks: write # enricomi/publish-unit-test-result-action
```

Comment:
Same question as in one of the other comments - do we need htis here? This is not a pipeline that runs Pester tests

Reply:
Removed.